cli/scale.go

```diff
@@ -0,0 +1,54 @@
+package cli
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+)
+
+var errInvalidScaleArg = errors.New("invalid argument")
+
+// This is a direct copy paste from
+// https://github.com/remind101/emp/blob/6fc34ef0633f5a26217023ede86b866bf4f17cc4/scale.go#L127-L168.
+//
```

Comment:
That's definitely the end goal. If https://github.com/remind101/empire/pull/712 gets merged then this can be shared.